packages/fxa-graphql-api/src/gql/dto/input/recovery-key.ts

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class RecoveryKeyBundleInput {
+  @Field({ description: 'Account reset token' })
+  public accountResetToken!: string;
```

Comment:
I am fine with this being a string, but in the auth-client it appears to be a hexstring.